src/index.js

```diff
@@ -1,9 +1,5 @@
 import './style.css';
 
-import { createTaskElement, deleteTaskElement, updateTaskText } from './adddelupd.js';
-
-import { updateTaskStatus, deleteCompletedTasks } from './completed.js';
-
 let tasksLocal = [];
```

Comment:
Kindy, avoid using global variables: The tasksLocal variable is declared outside the functions and is used as a global variable. This can lead to potential issues and conflicts in larger applications. Consider encapsulating the logic inside a class or module to avoid global variables.